.github/workflows/claude-nl-suite.yml

```
@@ -1,134 +1,351 @@
+  UNITY_VERSION: 2021.3.45f1
+  UNITY_IMAGE: unityci/editor:ubuntu-2021.3.45f1-linux-il2cpp-3
+  UNITY_CACHE_ROOT: /home/runner/work/_temp/_github_home
+
 jobs:
   nl-suite:
     if: github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     timeout-minutes: 60
+    env:
+      JUNIT_OUT: reports/junit.xml
+      MD_OUT:    reports/junit-nl-suite.md
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)